setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -1,8 +1,5 @@
 use vs
 
-package name=Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles
-        version=$(Version)
-
```

Comment:
This information is now auto-injected as part of the process that creates the VSIX package.